tests/end_to_end/test_client_json.py

```diff
@@ -73,3 +73,7 @@ def testGetDataset(self):
         for dataset in self._client.searchDatasets():
             self.verifyParsedOutputsEqual(
                 [dataset], "datasets-get", dataset.id)
+
+    def testSearchAllReadGroups(self):
+        cliOutput = self.captureJsonOutput("reads-search")
+        self.assertGreater(len(cliOutput), 0)
```

Comment:
there's probably a more rigorous test that we can write here... if the output is not what we expect, this test still passes